src/cpp/rtps/transport/UDPv4Transport.cpp

```diff
@@ -393,10 +393,6 @@ eProsimaUDPSocket UDPv4Transport::OpenAndBindInputSocket(
 {
     eProsimaUDPSocket socket = createUDPSocket(io_service_);
     getSocketPtr(socket)->open(generate_protocol());
-    if (mReceiveBufferSize != 0)
```

Comment:
I think in case `try_setting_receive_buffer_size` exists on `return minimum_buffer_value;` we have not tried to actually set `minimum_buffer_value`. We'd need to do that at some point, and probably return with failure if we can't. Same for send buffer and udpv6